tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -0,0 +1,313 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+from decimal import Decimal
+from tempfile import mkdtemp
+from typing import Any, Tuple
+
+import pytest
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col
+from pyspark.sql.types import DecimalType, IntegerType, StringType, StructField, StructType
+
+from streaming import MDSWriter
+from streaming.base.converters import dataframeToMDS
+
+MY_PREFIX = 'train'
+MY_BUCKET = 'mosaicml-composer-tests'
+LOCAL_MANUAL_TEST = False
+os.environ[
+    'OBJC_DISABLE_INITIALIZE_FORK_SAFETY'] = 'YES'  # set to yes to all fork process in spark calls
+
+
+@pytest.fixture(scope='class', autouse=True)
+def remote_local_dir() -> Any:
+    """Creates a temporary directory and then deletes it when the calling function is done."""
+
+    os.environ[
+        'GOOGLE_APPLICATION_CREDENTIALS'] = '~/.mosaic/mosaicml-research-nonprod-027345ddbdfd.json'
```

Comment:
Can we have some dummy path here ? I am sure there is no credentials in this path!

Reply:
Done.